packages/survey-creator-core/src/components/item-value.scss

```diff
@@ -257,13 +257,13 @@
 
   .svc-question__content--ranking,
   .sd-selectbase {
-    padding-left: 0;
-    margin-left: 0;
-    overflow: initial;
+    // padding-left: 0;
```

Comment:
You promised to remove the comments later :-)

Reply:
https://github.com/surveyjs/survey-creator/commit/cbee3de0ef1ef6391fce8e2b1bd4c42a58350d55